Windows/Debugger/CtrlDisAsmView.cpp

```diff
@@ -1352,14 +1355,176 @@ void CtrlDisAsmView::disassembleToFile() {
 	}
 }
 
+
+// The following member has been modified to accomodate "nicer-looking" output (according to my needs). Assumes maximum length of an instruction name is 9 or 10 chars.
+// What's the maximum length the params string could reasonably be?
 void CtrlDisAsmView::getOpcodeText(u32 address, char* dest, int bufsize)
 {
 	DisassemblyLineInfo line;
 	address = manager.getStartAddress(address);
-	manager.getLine(address,displaySymbols,line);
-	snprintf(dest, bufsize, "%s  %s",line.name.c_str(),line.params.c_str());
+	manager.getLine(address, displaySymbols, line);
+	
+	// Begin trace logger mods
+	std::string instructionName = std::string(line.name.c_str());
```

Comment:
`line.name` is a `std::string` already, no reason to get a `const char *`, construct a `std::string` out of it (elided by the compiler likely) and use it to construct `instructionName`. 

`c_str()` is useful mostly when you are interacting with C style method and function (e.g. printf) that require `const char *`.

TL;DR `std::string instructionName = line.name;` should do.

Reply:
I think this was literally some of the first code i wrote so yeah i can clean this up.